addons/luamodel/CMakeLists.txt

```diff
@@ -10,16 +10,19 @@ FIND_PACKAGE (Lua 5.1 REQUIRED)
 
 INCLUDE_DIRECTORIES ( 
   ${CMAKE_CURRENT_BINARY_DIR}/include/rbdl
-  ${LUA_INCLUDE_DIR}
+  ${LUA_INCLUDE_DIR}  
```

Comment:
There is unnecessary trailing whitespace here that should be removed prior to merging.